specs/signature-specification.md

```diff
@@ -1,60 +1,18 @@
 # Signature Specification
 
-This document provides the following details for Notary v2 signatures:
+This document provides the following details for Notary Project signature:
 
 - **[Storage](#storage)**: Describes how signatures are stored and retrieved from an OCI registry.
-- **[Signature Envelope](#signature-envelope)**: Describes the structure of a Notary v2 signature.
+- **[Signature Envelope](#signature-envelope)**: Describes the structure of the Notary Project signature.
 
 ## Storage
 
-This section describes how Notary v2 signatures are stored in the OCI Distribution conformant registry.
-Notary v2 uses [OCI artifact manifest][oci-artifact-manifest] to store signatures in the registry.
-The media type of the signature manifest is `application/vnd.oci.artifact.manifest.v1+json`.
-The signature manifest has an artifact type that specifies it's a Notary V2 signature, a reference to the manifest of the artifact being signed, a blob referencing the signature, and a collection of annotations.
+This section describes how a Notary Project signature is stored in an OCI Distribution conformant registry.
+OCI image manifest is used to store signatures in the registry, see [OCI image spec v1.1.0-rc3][oci-image-manifest] for details.
```

Comment:
Need to create an issue for `v1.1.0-rc4` update.

Reply:
Done. See https://github.com/notaryproject/notaryproject/issues/270